### PR TITLE
Prepare releases, remove unused dependency

### PR DESCRIPTION
**Pull request by Thoralf-M**
Waiting at least for https://github.com/iotaledger/iota-sdk/pull/639, maybe more changes

## Links to any relevant issues

https://github.com/iotaledger/iota-sdk/issues/396

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Running `yarn build:nodejs` and `node examples/node.js`

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/python/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "iota-sdk-python"
-version = "0.1.0"
+version = "1.0.0-rc.1"
```
Comment (thibault-martinez) [inline bindings/python/Cargo.toml]: We started nodejs with rc.0

Comment (Thoralf-M) [reply bindings/python/Cargo.toml]: Yes but that seemed weird to me, https://semver.org/spec/v2.0.0-rc.1.html starts with 1 :thinking:
But this doesn't seem to be defined anywhere, so if you want I can change to 0

Comment (Thoralf-M) [reply bindings/python/Cargo.toml]: Updated

**Comment by thibault-martinez** [comment]
wen

**Review by kwek20 — APPROVED** [review]

**Review by qrayven — APPROVED** [review]

File: bindings/wasm/CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
-## 0.1.0 - 2023-MM-DD
+## 1.0.0-rc.0 - 2023-MM-DD
```
Comment (thibault-martinez) [inline bindings/wasm/CHANGELOG.md:22]: date wasn't changed

Comment (Thoralf-M) [reply bindings/wasm/CHANGELOG.md:22]: Ah yeah sorry, changed it in the wrong readme, Python should stay without exact date

**Pull request merged**